lumigator/frontend/src/stores/datasetsStore.ts

```diff
@@ -56,6 +55,9 @@ export const useDatasetStore = defineStore('datasets', () => {
     }
     inferenceJobs.value = allJobs
       .filter((job) => job.metadata.job_type === 'inference')
+      // NOTE: 'temporary fix' to prevent showing inference jobs that weren't created by the UI
```

Comment:
maybe filter jobs that don't have a set experiment ref (ref eq to null)?

Reply:
This was just a temporary fix for now but was part of: https://github.com/mozilla-ai/lumigator/pull/986